tests/PHPStan/Rules/Functions/data/call-to-weird-functions.php

```diff
@@ -39,3 +39,5 @@
 openssl_x509_parse('foo'); // OK
 openssl_x509_parse('foo', true); // OK
 openssl_x509_parse('foo', true, 'bar'); // should report 3 parameters given, 1-2 required
+
+get_defined_functions(); // OK for PHP <7.1.10
```

Comment:
Please also add a case that reports an error, like with the functions above.

Reply:
@ondrejmirtes If we add a test like:
```php
get_defined_functions(true, false);
```
We would have to situations to check:
- `Should report 2 parameters given, 0 required` for PHP <7.1.10
- `Should report 2 parameters given, 0-1 required` for PHP >7.1.10

Reply:
Oh I see, never mind, not worth the effort.